dbtmetabase/_exposures.py

```diff
@@ -112,7 +112,9 @@ def extract_exposures(
                 depends = set()
                 native_query = ""
                 header = ""
-
+                average_query_time_ms = None
+                last_used_at_utc = None
```

Comment:
Since you're retrieving them as-is from Metabase, keep the naming identical (i.e. no `_ms` and `_at_utc`)